src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ExportClient/OtlpRetry.cs

```diff
@@ -189,9 +194,14 @@ private static bool IsGrpcStatusCodeRetryable(StatusCode statusCode, bool hasRet
     }
 
 #pragma warning disable SA1313 // Parameter should begin with lower-case letter
-    private static bool IsHttpStatusCodeRetryable(HttpStatusCode statusCode, bool _)
+    private static bool IsHttpStatusCodeRetryable(HttpStatusCode? statusCode, bool _)
```

Comment:
nit: What is with this `bool _`? Private method could we just remove that? Or does this need to match some delegate signature?

Reply:
Yes: https://github.com/open-telemetry/opentelemetry-dotnet/blob/77ef12327f720ca3defd0c9590c0197cceb5952e/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ExportClient/OtlpRetry.cs#L65

Reply:
Got it. I will say, as-is it is super hard to read/understand. Consider just naming the thing:

```diff
-#pragma warning disable SA1313 // Parameter should begin with lower-case letter
-    private static bool IsHttpStatusCodeRetryable(HttpStatusCode statusCode, bool _)
-#pragma warning restore SA1313 // Parameter should begin with lower-case letter
+private static bool IsHttpStatusCodeRetryable(HttpStatusCode statusCode, bool hasRetryDelay)
```

I don't _think_ you'll get a warning/suggestion about "hasRetryDelay" not being used because the compiler knows it is needed to satisfy the delegate contract.